cli/cmd/capture/capture.go

```diff
@@ -20,7 +23,8 @@ var capture = &cobra.Command{
 
 func init() {
 	cmd.Retina.AddCommand(capture)
-	configFlags = genericclioptions.NewConfigFlags(true)
-	configFlags.AddFlags(capture.PersistentFlags())
-	capture.PersistentFlags().StringVar(&name, "name", defaultName, "The name of the Retina Capture")
+	opts.ConfigFlags = *genericclioptions.NewConfigFlags(true)
+	opts.AddFlags(capture.PersistentFlags())
+	capture.PersistentFlags().StringVar(opts.Name, "name", defaultName, "The name of the Retina Capture")
+	_ = capture.MarkPersistentFlagRequired("name")
```

Comment:
Do we need to mark `name` as a required flag? I previously got rid of this considering we have a default value and the capture file names are unique because of the timestamp.

Reply:
It snuck back in when I rebase/resolved conflicts. Removed